modules/help_editor/js/help_editor_helper.js

```diff
@@ -1,5 +1,3 @@
-import swal from 'sweetalert2';
```

Comment:
why is this downgrading from swal2 to swal1?

Reply:
When @zaliqarosli was making the changes, she said that this statement threw an error. However, when I added this line back on my local version, I did not get the same error. I'm not sure why this was causing issues for her but not me.

Reply:
I don't think it should be downgraded.

Reply:
undoing the changes on this file is still giving me the error:

<img width="668" alt="Screen Shot 2020-09-21 at 1 16 02 PM" src="https://user-images.githubusercontent.com/17415878/93799147-beda3500-fc0c-11ea-82bc-8012399caacb.png">

@laemtl also reported seeing this error in https://github.com/aces/Loris/issues/6940. (Laetitia, any chance you could give this PR a test, with and without these changes to this file?)

@driusan @HenriRabalais @maltheism any of you know how to properly fix this without downgrading to swal1?

Reply:
Yes sure !

Reply:
The reason swal2 isn't accessible is because the help_editor_helper.js is inside the `js` directory and not the `jsx` directory that webpack handles. I think she could move the file to the jsx directory while maybe needing to add it to webpack where the other file the help_editor module uses.

Reply:
@zaliqarosli You are right, there is a bug introduced by #6195. The migration to swal2 introduced problems.

Reply:
@laemtl see htdocs/vendor/sweetalert for the old sweetalert.js that gets accessed anywhere in Loris. I think if we placed the `node_modules/sweetalert/dist/sweetalert2.js` inside that directory or a new one. Then swal2 would be accessible without needing to import.

Reply:
I would like to fix this in a compatible way with my PR #7018. I did fix this for another script by reactifying it.

Reply:
@driusan how would you prefer us to proceed?

Reply:
i think this PR should just fix the bug introduced in https://github.com/aces/Loris/pull/6195/files#diff-319f29c33167f3c507dcb5516ec2578c and the move to sweetalert2 should be tackled in another PR

Reply:
I'm not sure I understand why the file can't just be moved from the `js` directory to `jsx` and added to the webpack config so that webpack compiles it and it can use swal2?

Reply:
@driusan Eslint fails with /var/www/html/loris/modules/help_editor/jsx/help_editor_helper.js
33:13  error  A function with a name starting with an uppercase letter should only be used as a constructor  new-cap


```
markdownContent = document.createElement('div');
ReactDOM.render(
       RMarkdown({content: content}),
       markdownContent
);
```

Reply:
wouldn't it be kind of against our repo standard that we're calling a non-react non-jsx file a jsx file and putting it under that directory? help_editor_helper.js isn't a component and doesn't need to be compiled. there's a way to call sweetalert2 with normal JS:

<img width="821" alt="Screen Shot 2020-09-21 at 5 37 34 PM" src="https://user-images.githubusercontent.com/17415878/93824384-610c1400-fc31-11ea-9c86-0e2658e62edf.png">

but using require() also throws an error. surely just making the file a jsx file isn't the right move?

Reply:
@zaliqarosli nodejs, browserify, or webpack supports require() but not javascript on the browser. The browser shouldn't have access to node_modules on the server anyways.

Reply:
@zaliqarosli You are right, I migrated my fix to #7018.